overlap-scripts/kubectl-kubestellar-ensure-wmw

```diff
@@ -77,19 +77,33 @@ then kubectl ws "${kubectl_flags[@]}" "$wmw_name"
 else kubectl ws "${kubectl_flags[@]}" create "$wmw_name" --enter
 fi
 
-if ! kubectl "${kubectl_flags[@]}" get APIBinding bind-espw &> /dev/null; then
-kubectl "${kubectl_flags[@]}" apply -f - <<EOF
-apiVersion: apis.kcp.io/v1alpha1
-kind: APIBinding
-metadata:
-  name: bind-espw
-spec:
-  reference:
-    export:
-      path: root:espw
-      name: edge.kubestellar.io
-EOF
-fi
+function run_kb_konnector_for_wmw() {
+    if kubectl get ns kube-system &> /dev/null; then
+        echo "namespace kube-system already exists, skipping"
+    else
+        echo "creating namespace kube-system"
+        kubectl create ns kube-system
+    fi
+    echo "starting kube-bind konnector for wmw in background"
+    konnector &> /dev/null &
+    if ! pgrep konnector &> /dev/null; then
+        echo "kube-bind konnector not started"
+        exit 1
+    fi
+}
+
+function bind_resource_for_wmw() {
+    RESOURCE="$1"
+    echo "binding $RESOURCE for wmw"
+    if ! kubectl bind http://127.0.0.1:8080/export --skip-konnector --unattended --resource "$RESOURCE" ; then
```

Comment:
`127.0.0.1:8080` seems too specific to me. This script needs to be something that a user can invoke in their own shell, even if the core controllers are hosted in a kube cluster.

Reply:
My understanding is that the plan is to focus at first on making `kubectl kubestellar ensure wmw` work inside the core container and fix the problems with using it outside the core container in follow-on work.